### DISCUSSION TITLE
[@types/wordpress__notices] Incorrect CallbackAction types

### DISCUSSION THREAD

**Discussion by chihsuan** [DefinitelyTyped/DefinitelyTyped#58850] [Issues with a @types package]
For`WPNoticeAction`, the `callback` has been changed to [`onClick`](https://github.com/WordPress/gutenberg/blob/04ba33ee9585e7864e241313146526055e68ab2a/packages/notices/src/store/actions.js#L17).  However, [`@types/wordpress__notices`]((https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wordpress__notices/index.d.ts#L32)) still uses `callback`.  

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a96f8c2be7ae8a465a2a06c6a2d34b23e6ea1b4d/types/wordpress__notices/index.d.ts#L32

This is causing a type error:

![Screen Shot 2022-02-17 at 16 45 44](https://user-images.githubusercontent.com/4344253/154438792-ea2cae18-d4f8-44ce-8d8e-c966e0b3de87.png)

[a comment by an automated account is not shown]

**Comment by chihsuan** [comment]
cc @sirreal @dmsnell

**Comment by sirreal** [comment]
Thanks @chihsuan, do you mind opening a PR to update the types?

**Reply by chihsuan** [reply]
Sure. l’ll do it later!

**Reply by chihsuan** [reply]
Update: PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58884
